File: Pemro1_Modul5/Bahasa_C/PRAK504-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>
int reverse(int x){
    int hasil = 0;
    while (x > 0) {
        hasil = hasil * 10 + (x % 10);
        x = x / 10;
    }
    return hasil;
}

int main() {
    int A, B;
    scanf("%d %d", &A, &B);

    A = reverse(A);
    B = reverse(B);

    int C = A + B;
    printf("%d", reverse(C));

    return 0;
}

File: Pemro1_Modul5/Bahasa_Python/PRAK504-2510817320010-PutriDeissytaHerliyana.py
def reverse(x):
    hasil = 0
    while x > 0:
        hasil = hasil * 10 + (x % 10)
        x //= 10
    return hasil

A, B = map(int, input().split())

A = reverse(A)
B = reverse(B)

C = A + B
print(reverse(C))
